gTea/app/src/main/java/com/nechaieva/gtea/MainActivity.java

```diff
@@ -25,18 +32,52 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
+        super.onCreateOptionsMenu(menu);
         getMenuInflater().inflate(R.menu.menu_main, menu);
+        /*View accountItem = menu.findItem(R.id.action_open_user_profile).getActionView();
```

Comment:
It is rarely advisable to add commented out code. Commented code raises lots of questions, like "why are those lines commented? Are they important?" (see clean code book)

Reply:
Same as above. It was only present during the WIP; see the latest version.